docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -527,15 +524,9 @@ The `$validate` operation checks whether the attached content would be acceptabl
 * If a profile is specified via the optional `profile` parameter, the $validate operation will validate a resource against the base specification and the specified profile only. It will not validate against any other profiles asserted in the `Resource.meta.profile` element.
 * If the `profile` parameter is not specified, but the `mode` parameter is specified, and the `mode` parameter value is either `create` or `update`, the $validate operation will validate a resource against the base specification and any profiles asserted in its `Resource.meta.profile` element, but will do so based on profile configuration properties specified in the `fhirServer/resources/<resourceType>/profiles` section of the `fhir-server-config.json` file (see the [Configuration properties reference](#51-configuration-properties-reference) for configuration details).
 
+https://hl7.org/fhir/R4B/resource-operation-validate.html
 
-https://www.hl7.org/fhir/r4/resource-operations.html#validate
-
-#### 4.1.1.2 $document
-The `$document` operation generates a fully bundled document from a composition resource.
```

Comment:
IMHO it didn't make sense to highlight the $document operation here.  We implement a whole host of operations and for some reason this was one of the few that we put here.  Rather than add sections for each one, I removed this one.  I debated removing the $healthcheck section too, but at least that operation is useful...so I kept it for now.